src/components/Agents/InstallAgentModal.tsx

```diff
@@ -0,0 +1,39 @@
+import { GeneratedAgent } from "../../api/services/agents";
+import { Button } from "../Button";
+import CodeBlock from "../CodeBlock/CodeBlock";
+import { Modal } from "../Modal";
+
+type Props = {
+  isOpen: boolean;
+  onClose: () => void;
+  generatedAgent: GeneratedAgent;
+};
+
+export default function InstallAgentModal({
+  isOpen,
+  onClose,
+  generatedAgent
+}: Props) {
+  return (
+    <Modal
+      title={"Instructions to install the agent"}
+      onClose={onClose}
+      open={isOpen}
+      bodyClass="flex flex-col w-full flex-1 h-full overflow-y-auto"
+    >
+      <div className="flex flex-col flex-1 p-4 gap-4 overflow-y-auto">
+        <p>1. Add the Flanksource Helm repository to your local Helm client</p>
+        <CodeBlock code="helm repo add flanksource https://flanksource.github.io/charts" />
+        <p>2. And run repo update </p>
+        <CodeBlock code={"helm repo update"} />
+        <p>3. And finally install the agent with the following command.</p>
+        <CodeBlock
+          code={`helm install mc-agent flanksource/mission-control-agent -n "<namespace>"  --create-namespace --set upstream.createSecret=true --set upstream.host=$host --set upstream.username=${generatedAgent.username} --set upstream.password=${generatedAgent.access_token}`}
```

Comment:
The host needs to come from the org metadata (or the current location)

Namespace should also be "mission-control-agent" not `<namesspace>`